src/types/process.go

```diff
@@ -16,6 +16,8 @@ type ProcessConfig struct {
 	Disabled          bool                   `yaml:"disabled,omitempty"`
 	IsDaemon          bool                   `yaml:"is_daemon,omitempty"`
 	Command           string                 `yaml:"command"`
```

Comment:
Should args and command (with more than one argument) be mutually exclusive?

Reply:
Personally I’d live it to the user. There is an odd chance that an executable has a space in it.

Reply:
👍🏻